### ISSUE TITLE
import more than one GLB

### ISSUE THREAD

**Issue by undefinederror** [BabylonJS/Editor#245]
I am having serious problems with importing multiple GLBs
I import one GLB, materials seem ok, texture too.
I import a second GLB. Materials are the same as first GLB, same goes for textures.
Same happens with all subsequent meshes

![image](https://user-images.githubusercontent.com/8209966/103282384-e4c66100-49d5-11eb-81e8-5e1351009658.png)
![image](https://user-images.githubusercontent.com/8209966/103282393-edb73280-49d5-11eb-9daf-0d9ca662df9e.png)

If I remove the nodes in scene. Clear unused, then import a GLB, it just shows materials and textures from the first ever GLB I imported. Same goes for File => Reload Project, it doesn't clear cache.
File => Open Workspace does clear cache

Let me know if you need more info.
Thank you

**Comment by undefinederror** [comment]
No,  File => Open Workspace doesn't clear cache either.
Only creating a new project does.
So once a GLB has been created, materials and texture for all subsequent imports won't change.
I am talking GLB because that's what I am using right now. I suspect the same goes for other formats

**Comment by julien-moreau** [comment]
Hey @undefinederror I'm checking right now if the GLTF loader checks for existing materials and textures before creating them. If it's not the case, we'll report that issue on the Babylon.JS repository.

To be sure I have understood, are you importing the same GLB file twice ? Or are you importing distinct GLB files that share exactly same materials ?

**Comment by undefinederror** [comment]
Hi @julien-moreau ,
Thank you for your reply.

I am importing different GLBs that share the same materials names. 
For the the moment I had to resort to not using the editor because I have a presentation next week and really need to get as much done as quickly as possible.

Let me clarify a couple of things. In my GLBs the material names are the same, but I am not sure why. Whether they are meant to be the same vs totally different materials that happen to have same names. I will know that on Thursday.

In my code I am using `SceneLoader.LoadAssetContainerAsync`, for each GLB, then I am instantiating each mesh into the scene. This allows to have different materials with same name yet with different textures and property values.
I believe that is because the meshes that are included in the scene have a reference to the materials in their container, rather than the scene.

Having set up my app this way from the beginning, I thought this was standard behaviour, As I approached the editor and noticed a different behaviour I thought this was a bug, but maybe it's just due to a different method used by SceneLoader to load assets?
